v0/destinations/algolia/transform.js

```diff
@@ -92,17 +126,31 @@ const trackResponseBuilder = (message, { Config }) => {
 const process = event => {
   const { message, destination } = event;
   if (!message.type) {
-    throw new CustomError(
```

Comment:
The error object properties are different than before.
```
// prev
{
  message: "message Type is not present. Aborting message.",
  response: {
    status: 400
  }
}

// current
{
  message: "message Type is not present. Aborting message.",
  status: 400,
  statTags: {
  // the tags we set
  }
}
```

Are we okay with it?

Reply:
By okay, if you meant that this change would cause some kind of error on server. That is not the case

Reply:
Either of these formats must fail due to the data contract breach, right?

Reply:
In the `processRouterDest` method's try-catch block(inside the `.map`) we do something like this.

```javascript
// Ref: https://github.com/rudderlabs/rudder-transformer/blob/master/v0/util/index.js#L1589-L1612
const errRespEvent = handleRtTfSingleEventError(
          input,
          error,
          "ALGOLIA"
        );
```

This will make sure that, we adhere to the contract. Let me know if it makes sense